src/Microsoft.Extensions.FileProviders.Embedded/EmbeddedFileProvider.cs

```diff
@@ -49,8 +49,15 @@ public EmbeddedFileProvider(Assembly assembly, string baseNamespace)
 
             _baseNamespace = string.IsNullOrEmpty(baseNamespace) ? string.Empty : baseNamespace + ".";
             _assembly = assembly;
-            // REVIEW: Does this even make sense?
-            _lastModified = DateTimeOffset.MaxValue;
+
+            _lastModified = DateTimeOffset.UtcNow;
+
+#if NETSTANDARD1_5
+            if (_assembly.Location != null)
+            {
+                _lastModified = File.GetLastWriteTimeUtc(_assembly.Location);
```

Comment:
No harm in try catching this? On a bit of a tangent, did we consider having the alternative where the user specifies this value? It removes a bunch of this guessing from our code (plus the oddity with having different behavior based on what you might be targeting)?